garrysmod/lua/includes/extensions/player.lua

```diff
@@ -176,7 +176,7 @@ end
 -----------------------------------------------------------]]
 function meta:GetPData( name, default )
 
-	name = Format( "%s[%s]", self:UniqueID(), name )
+	name = Format( "%s[%s]", self:SteamID64(), name )

```

Comment:
This will remove the ability for any current sv/cl.db databases to be read.

Reply:
Yes it is a "poor reason" if your server never had such collision.
It is not a "poor reason" when there are 2 players with different SteamIDs but their UniqueIDs being the same in the server database (PData). For example: they would have same inventories.
This is "poor reason" to you?
This whole patch is just making me sick..

Reply:
The collisions are a serious issue, but you put in no effort to maintaining backwards compatibility or database migration. This whole patch seems like a hasty solution with little forethought.